utils/docker/Dockerfile.centos.7

```
@@ -94,12 +94,10 @@ RUN useradd -u $UID -ms /bin/bash $USER
 RUN echo "$USER:$PASSWD" | chpasswd
 
 # Create directory for DAOS backend storage
-RUN mkdir /mnt/daos
-RUN chown daos.daos /mnt/daos
-RUN mkdir /var/run/daos_server
-RUN chown daos.daos /var/run/daos_server
-RUN mkdir /var/run/daos_agent
-RUN chown daos.daos /var/run/daos_agent
+RUN mkdir -p /opt/daos && chown -R daos.daos /opt/daos
+RUN mkdir /mnt/daos && chown daos.daos /mnt/daos
+RUN mkdir /var/run/daos_server && chown daos.daos /var/run/daos_server
+RUN mkdir /var/run/daos_agent && chown daos.daos /var/run/daos_agent
```

Something to think about in the future:
```suggestion
RUN for dir in /opt/daos /mnt/daos /var/run/daos_server /var/run/daos_agent; do \
        mkdir $dir && chown daos.daos $dir; \
    done
```